hugegraph-server/hugegraph-dist/src/assembly/static/bin/dump-store.sh

```
@@ -30,22 +30,26 @@ TOP="$(cd $BIN/../ && pwd)"
 else
     JAVA=java
 fi
 
-conf=$1
 if [ $# -eq 0 ]; then
-    conf=$CONF/hugegraph.properties
+    set -- "$CONF/graphs/hugegraph.properties"
 fi
+conf=$1
 
 cd $TOP
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)